### DISCUSSION TITLE
Does radix has a figma that designers can use?

### DISCUSSION THREAD

**Discussion by PezCoder** [radix-ui/primitives#1816] [Help] [answered]
Hey! We're starting a new project & a part of our evaluation (& need) is the figma support for the base component library that we decide.

The reason being designers communicate their ideas through a design tool (ex: figma in our case) & radix is the base of the components for us, on which we'll build pages/websites, hence this is an essential requirement for us.

Here's an example of some common libraries which does it:
- [Uber's base web](https://baseweb.design/) - https://www.figma.com/community/file/805195278314519508
- [Antd library](https://ant.design/docs/react/introduce) - https://www.figma.com/community/file/831698976089873405 ⭐ 

Related to Radix this is the only one I could find:
https://www.figma.com/community/file/1055912623219285224

Which seems unofficial & has a subset of components only.

I wanted to know if there is already a library out there, or a plan anytime soon to export these primitives within a design tool (ex: figma)?

**Comment by joaom00** [comment, marked as answer]
Hey @PezCoder,

Radix primitives is an unstyled component library, so I guess there must be no design file.
